lib/rules/selector-class-pattern/__tests__/index.mjs

```diff
@@ -182,6 +235,39 @@ testRule({
 			endLine: 1,
 			endColumn: 11,
 		},
+		{
+			code: stripIndent`
```

Comment:
The logic of `resolveNestedSelectors` wasn't changed, but confirmed to work.

<img width="583" alt="image" src="https://github.com/user-attachments/assets/0f837e2c-64c4-4317-8390-0b4cee5b7b50">